_visual/typography.md

```diff
@@ -13,8 +12,7 @@ Include intro of the two families, how we have diff packages, a bit on considera
 
 <div class="usa-grid-full">
   <div class="usa-width-one-half">
-    <p>Explanation of font selection Explanation of font selection Explanation of font selection. Explanation of font selection Explanation of font selection Explanation of font selection.</p>
-    <p>Explanation of font selection Explanation of font selection. Explanation.</p>
+    <p>Source Sans Pro is an open-source sans serif typeface created for legibility in UI design.  With a variety of weights that read easily at all sizes, Source Sans Pro provides clear headers as well as highly readable body text.  Its slender but open letters offer a clean and friendly simplicity. Advanced hinting allows Source Sans Pro to render well on Windows systems which run Cleartype, and across browsers and devices. Moreover, it supports a variety of languages and alphabets, including Western and European language, Vietnamese, pinyin Romanization of Chinese, and Navajo.</p>
```

Comment:
I would love to include something about this font being inspired by twentieth-century American gothics, which became a staple of almost every type foundry coming after. It's a very American thing we can highlight.

"Source Sans Pro draws inspiration from the clarity and legibility of twentieth-century American gothic typeface designs. Distilling the best archetypical qualities of these models, Paul followed a rational design approach by simplifying glyph shapes by paring them to their essential form. "

http://www.fontsquirrel.com/fonts/source-sans-pro

cc @mollieru